simulation/traffic_simulator/include/traffic_simulator/entity/pedestrian_entity.hpp

```diff
@@ -98,6 +98,10 @@ class PedestrianEntity : public EntityBase
     return parameters.bounding_box;
   }
 
+  auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel;
```

Comment:
```
auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel;
```
↓
```
auto getDriverModel() const -> traffic_simulator_msgs::msg::DriverModel;
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/650/commits/d3a8c09835cca7d0b623b86317f54f9152eb257e